### PR TITLE
Fix dataplane pre-commit

### PR DESCRIPTION
**Pull request by fao89**
https://github.com/openstack-k8s-operators/dataplane-operator/pull/655

https://github.com/openstack-k8s-operators/openstack-ansibleee-operator/pull/312

### CODE REVIEW

**Comment by fao89** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-dataplane-operator-main-precommit-check

**Comment by fao89** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-dataplane-operator-main-precommit-check

**Comment by fao89** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-dataplane-operator-main-precommit-check

**Comment by fao89** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-dataplane-operator-main-precommit-check

**Comment by fao89** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-dataplane-operator-main-precommit-check

**Comment by fao89** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-dataplane-operator-main-precommit-check

**Comment by fao89** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-dataplane-operator-main-precommit-check

File: ci-operator/config/openstack-k8s-operators/openstack-ansibleee-operator/openstack-k8s-operators-openstack-ansibleee-operator-main.yaml [hunk 1]
```diff
@@ -68,7 +69,7 @@ tests:
     git log -1
     mkdir -p ../operator && cp -r . ../operator
     cd ../operator
-    pre-commit run --all-files --show-diff-on-failure
+    BUNDLE_PATH="./local/bundle" pre-commit run --all-files --show-diff-on-failure
```
Comment (rabi) [inline ci-operator/config/openstack-k8s-operators/openstack-ansibleee-operator/openstack-k8s-operators-openstack-ansibleee-operator-main.yaml:72]: Should not this be `/usr/bin/bundle` ?
Error says `Bundler not found. On Linux run 'sudo dnf install /usr/bin/bundle' to install it.`

Comment (fao89) [reply ci-operator/config/openstack-k8s-operators/openstack-ansibleee-operator/openstack-k8s-operators-openstack-ansibleee-operator-main.yaml:72]: actually, `/usr/bin/bundle` is for installer the bundler, but it depends on which ruby version you get.
by `dnf install -y rubygem-bundler` I'm ensuring the bundler is installed

Comment (rabi) [reply ci-operator/config/openstack-k8s-operators/openstack-ansibleee-operator/openstack-k8s-operators-openstack-ansibleee-operator-main.yaml:72]: Ok, $BUNDLE_PATH is the location on disk where all gems in your bundle will be located. I've forgotten ruby bits now :smile:

**Review by rabi — APPROVED** [review]
/lgtm

**Pull request merged**